apps/webpack.js

```diff
@@ -44,6 +44,8 @@ var baseConfig = {
         include: [
           path.resolve(__dirname, 'static'),
           path.resolve(__dirname, 'src'),
+          path.resolve(__dirname, 'test'),
+          path.resolve(`${__dirname}/../dashboard/app/assets/`, 'images'),
```

Comment:
I'm not actually sure how well advised this is. It feels pretty dirty having apps needing to reach into dashboard to do a build. @pcardune May also have opinions.